src/internal/segment/builder.cpp

```diff
@@ -114,7 +114,7 @@ void Builder::add_object(const std::string& name, std::shared_ptr<::srf::segment
     m_objects[name] = std::move(object);
 }
 
-void Builder::add_runnable(const std::string& name, std::shared_ptr<runnable::Launchable> runnable)
+void Builder::add_runnable(const std::string& name, std::shared_ptr<srf::runnable::Launchable> runnable)
```

Comment:
Why does this need to be fully qualified?

Reply:
If you are in srf::internal::xx and there is srf::xx, the you need to fully quality objects from the public NS.

Reply:
We need to fix up these duplicate names